### PR TITLE
Draft minutes for 10/07/19 Packaging WG Meeting

### PR DESCRIPTION
**Pull request by drbenmorgan**
Taken from Live Notes, minor spelling/format/grammar updates

### CODE REVIEW

**Comment by drbenmorgan** [comment]
Travis error is the usual htmlproofer link timeout, but the offending link "https://competitions.codalab.org/competitions/20112" does appear to be dead (at the time of writing ay least).

File: organization/_posts/2019/2019-07-10-packaging.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      System/XQuartz OpenGL. Will report to Spack GitHub/Slack when diagnosed/fixed
+    - Looking at Environments/Views for runtime
+      - Hobbs: some issues found with views as they do not allow more general environment
+        settings like ROOTSYS and similar.
+      - Javier: Modules more flexible here, can provide environment specifications in recipe,
+        and resultant files used by other third-party module systems
+  - FNAL's MVP1a for LArSoft Spack/SPackDev released for testing and feedback
+    - Ben has tried the initial setup/install with success.
+- AIDA++ LoI for Turnkey Stack"
+  - In to AIDA call: https://docs.google.com/document/d/1wf9AnL2T713idpdG0qz8J2v6iPI94LZscY5PL333Ikc
```
Comment (graeme-a-stewart) [inline organization/_posts/2019/2019-07-10-packaging.md]: Maybe make this a link (add `<>`)

File: organization/_posts/2019/2019-07-10-packaging.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+- Presentation from FCC meeting
+  - More than just packaging, but of interest to cover larger problem space and motivate work of packaging
+  - Motivation is whole experiment software lifecycle
+  - First: ideas, to concepts, to design.
+    - Design needs flexibility, but lots of details to think about
+  - Then:
+    - Production
+      - Real world: calib, align, dead/noise
+      - Stability, but continually improve
+  - Upgrade
```
Comment (graeme-a-stewart) [inline organization/_posts/2019/2019-07-10-packaging.md]: Indent one more level?

File: organization/_posts/2019/2019-07-10-packaging.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    - Design and implement improvements
+  - Preservation
```
Comment (graeme-a-stewart) [inline organization/_posts/2019/2019-07-10-packaging.md]: Indent one more level?

**Comment by drbenmorgan** [comment]
Comments from @graeme-a-stewart applied, and checks are same as before, so self merging...

**Pull request merged**